<launch>


    <!-- Run the local trajectory planner -->
    <node pkg="auto_nav" type="pid_position_follow" name="pid_position_follow" output="screen" respawn="true">
    	<rosparam file="$(find auto_nav)/config/point_pid_params/track_follow_params.yaml"/>
    	
    	<param name="simulink" value="true" />
    	 	
        <param name="local_path_topic" value="/local_path" />
   		<param name="global_path_topic" value="/move_base/GlobalPlanner/plan" />
    	<param name="cmd_vel_topic" value="/cmd_vel" />
    	<param name="game_state_topic" value="/game_state" />
    	
    	<param name="gimbal_yaw_position_topic" value="/4ws_auto_infantry/yaw_steering_position_controller/command" />
   		<param name="gimbal_pitch_position_topic" value="/4ws_auto_infantry/pitch_steering_position_controller/command" />
    	<param name="jointstate_topic" value="/joint_states" />
    	
    </node>

    

</launch>
